front-end/package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "oddbox-frontend-technical-test",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.18.40",
+    "@types/react": "^18.2.19",
+    "@types/react-dom": "^18.2.7",
+    "concurrently": "^8.2.0",
+    "json-server": "^0.17.3",
+    "parse-link-header": "^2.0.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "typescript": "^4.9.5",
+    "web-vitals": "^2.1.4"
+  },
```

Fix dependency warning when running `npm run start`
```suggestion
  },
    "devDependencies": {
    "@babel/plugin-proposal-private-property-in-object": "7.21.11"
  },
```